kimchi/src/circuits/polynomials/permutation.rs

```diff
@@ -335,7 +340,11 @@ impl<F: PrimeField, G: KimchiCurve<ScalarField = F>> ProverIndex<G> {
         //~
         //~ $\text{scalar} \cdot \sigma_6(x)$
         //~
-        let zkpm_zeta = self.cs.precomputations().zkpm.evaluate(&zeta);
+        let zkpm_zeta = self
+            .cs
+            .precomputations()
+            .permutation_vanishing_polynomial_m
```

Comment:
I'm not sure I can easily get the difference between `permutation_vanishing_polynomial_m` and `permutation_vanishing_polynomial_l`. Do you think the `m` and `l` could be expanded in the identifier?

Reply:
The `m` suffix used to mean 'monomial', and the `l` 'lagrange', corresponding respectively to the coefficients of the polynomial (`[a0, a1, ...]` for `f(x) = a0 x^0 + a1 x^1 + ...`) and the evaluations at each element of the domain (`[f(omega^0), f(omega^1), ...]`).

If you don't mind, I'm going to leave this as is for now, and fold them into the `EvaluatedColumnCoefficients` and `ColumnEvaluations` structures in the prover index respectively.